src/sentry/api/utils.py

```diff
@@ -176,9 +178,10 @@ def generate_organization_url(org_slug: str) -> str:
     return org_url_template.replace("{hostname}", generate_organization_hostname(org_slug))
 
 
-def generate_region_url() -> str:
+def generate_region_url(region: str | None = None) -> str:
```

Comment:
Not seeing where this would be used, but if it's possible, I'd rather have this take the `Region` object as an argument than a string.